jobbergate_cli/main.py

```diff
@@ -160,7 +160,7 @@ def main(ctx, username, password):
 
     ctx.obj["token"] = decode_token_to_dict(JOBBERGATE_API_JWT_PATH.read_text())
 
-    ctx.obj = Api(user_id=ctx.obj["token"]["user_id"])
+    ctx.obj = Api(1)
```

Comment:
I'm interested to know what is going on here. Shouldn't this be the user id as returned from the token?

Reply:
This needs to be refactored in the future, but the Api instance should not need to know about the user_id, since it is inside the token in the backend part. The user id logic is now fully managed by the backend, this fixes the problems about using token from userX and just putting userY's id to create objects "owned by userY"

Reply:
Is this something worth fixing for this pr then?

Reply:
No, because this will maybe lead to some new bugs, the current code is "stable" and working.
That change is better to be done in a separated PR. This one is very large already